Jenkinsfile

```diff
@@ -153,13 +153,31 @@ pipeline {
       }
     }
 
-    stage('Push Images') {
+    stage('Push Images Internally') {
       when {
         branch 'master'
       }
 
       steps {
+        sh './bin/publish_internal'
+      }
+    }
+
+    stage('Push Images') {
+      agent { label 'releaser-v2' }
+      // Only run this stage when triggered by a tag
+      when { tag "v*" }
+      steps {
+        // The tag trigger sets TAG_NAME as an environment variable
+
+        // Clean up first
+        sh 'docker run -i --rm -v $PWD:/src -w /src alpine/git clean -fxd'
```

Comment:
Are these required? I think those might have been oddities of the parse-a-changelog pipeline

I'm not sure the `deleteDir` is needed below either


If these are needed, can we please add a comment explaining why we're adding them to this PR?

Reply:
Yeah maybe this isn't necessary

Reply:
I'm taking it out until I find out if there's a good reason for it